Content.Shared/_RMC14/Xenonids/Evolution/XenoEvolutionSystem.cs

```diff
@@ -370,11 +370,14 @@ private bool CanEvolvePopup(Entity<XenoEvolutionComponent> xeno, EntProtoId newX
             var existing = 0;
             var total = 0;
             var current = EntityQueryEnumerator<XenoComponent>();
-            while (current.MoveNext(out var existingComp))
+            while (current.MoveNext(out var uid, out var existingComp))
             {
                 if (existingComp.Hive != oldHive)
                     continue;
 
+                if (HasComp<IgnoreForHivePopComponent>(uid))
```

Comment:
We have XenoComponent.CountedForSlots for this, I just forgot to use it here

Reply:
Ah